example/src/main/mdoc/rationale.md

```diff
@@ -0,0 +1,42 @@
+# Rationale
```

Comment:
why is this here twice?

Reply:
This document is the source, the other one is generated out of this by mdoc. This document does not contain any code so it could be here only once but I like to keep it consistent and have all the sources at single place.